src/main/java/io/epirus/console/account/AccountManager.java

```diff
@@ -53,59 +54,123 @@ public static void main(final CliConfig config, final String[] args) throws IOEx
+                        Console.exitError("Could not retrieve token. Try again later.");
                     } else {
-                        System.out.println(tokenError);
+
+                        Console.exitError(tokenError);
                     }
                     return;
                 }
                 String token = responseJsonObj.get("token").getAsString();
                 config.setLoginToken(token);
+                config.save();
                 System.out.println(
                         "Account created successfully. You can now use Epirus Cloud. Please confirm your e-mail within 24 hours to continue using all features without interruption.");
             } else {
-                System.out.println("Account creation failed. Please try again later.");
+                Console.exitError("Account creation failed. Please try again later.");
             }
 
         } catch (IOException e) {
-            System.out.println("Could not connect to the server.\nReason:" + e.getMessage());
+            Console.exitError("Could not connect to the server.\nReason:" + e.getMessage());
         }
     }
 
-    protected final Response executeClientCall(Request newAccountRequest) throws IOException {
+    private final Response executeClientCall(Request newAccountRequest) throws IOException {
         return client.newCall(newAccountRequest).execute();
     }
 
-    protected final RequestBody createRequestBody(String email) {
+    final RequestBody createRequestBody(String email) {
 
         return new FormBody.Builder().add("email", email).build();
     }
 
-    protected final Request createRequest(RequestBody accountBody) {
+    final Request createRequest(RequestBody accountBody) {
 
         return new Request.Builder()
-                .url(String.format("%s/auth/realms/EpirusPortal/web3j-token/create", CLOUD_URL))
+                .url(String.format("%s/auth/realms/EpirusPortal/web3j-token/create", cloudURL))
                 .post(accountBody)
                 .build();
     }
 
+    public void checkIfAccountIsConfirmed() throws IOException, InterruptedException {
+        Request request =
+                new Request.Builder()
+                        .url(
+                                cloudURL
+                                        + "/auth/realms/EpirusPortal/web3j-token/status/"
+                                        + config.getLoginToken())
+                        .get()
+                        .build();
+        System.out.println("Checking if the account is activated...");
+        int tries = 5;
+        while (tries-- > 0) {
+            if (userConfirmedAccount(request)) {
+                System.out.println("Account is active.");
+                return;
+            } else {
+                Thread.sleep(5000);
+            }
+        }
+        Console.exitError(
+                "Please check your email and activate your account in order to take advantage our features.");
```

Comment:
What should the user do if this happens? They'd activate the account maybe a bit later, and then they can rerun the command? If so we should probably say that.

Reply:
Yes so if they activate their account later they still need to rerun the command. I need to make this more clear.

Reply:
Added additional information to this advising the user to activate their account and re-run the command.